modules/ROOT/pages/procedures.adoc

```diff
@@ -0,0 +1,2236 @@
+| `config` | `MAP` | A map containing a single key `maxInvocations` (default value of 100).
+.3+| *Return arguments* | *Name* | *Type* | *Description*
+| `section` | `STRING` | The section retrieved.
+| `data` | `MAP` | Data pertaining to the retrieved statistics.
+| *Mode* 3+| READ
+|===
+
+[role=label--admin-only]
+[[procedure_db_stats_retrieveallanonymized]]
+=== db.stats.retrieveAllAnonymized()
+
+
+.Details
+|===
+| *Syntax* 3+m| db.stats.retrieveAllAnonymized(graphToken [, config ]) :: (section, data)
+| *Description* 3+a| Retrieve all available statistical data about the current database, in an anonymized form.
+.3+| *Input arguments* | *Name* | *Type* | *Description*
+| `graphToken` | `STRING` | The name of the graph token.
+| `config` | `MAP` | A map containing a single key `maxInvocations` (default value of 100).
+.3+| *Return arguments* | *Name* | *Type* | *Description*
+| `section` | `STRING` | The section retrieved.
+| `data` | `MAP` | Data pertaining to the retrieved statistics.
+| *Mode* 3+| READ
+|===
+
+
+[role=label--admin-only]
+[[procedure_db_stats_status]]
+=== db.stats.status()
+
+
+.Details
+|===
+| *Syntax* 3+m| db.stats.status() :: (section, status, data)
+| *Description* 3+a| Retrieve the status of all available collector daemons, for this database.
+.4+| *Return arguments* | *Name* | *Type* | *Description*
+| `section` | `STRING` | String with the message "QUERIES".
+| `status` | `STRING` | The status of the QueryCollector: "idle" or "collecting".
+| `data` | `MAP` | data :: MAP
+| *Mode* 3+| READ
+|===
+
+
+[role=label--admin-only]
+[[procedure_db_stats_stop]]
+=== db.stats.stop()
+
+
+.Details
+|===
+| *Syntax* 3+m| db.stats.stop(section) :: (section, success, message)
+| *Description* 3+a| Stop data collection of a given data section. Valid sections are 'QUERIES'
+.2+| *Input arguments* | *Name* | *Type* | *Description*
+| `section` | `STRING` | Specify 'QUERIES'.
+.4+| *Return arguments* | *Name* | *Type* | *Description*
+| `section` | `STRING` | The stopped section.
+| `success` | `BOOLEAN` | Whether the section was successfully stopped.
+| `message` | `STRING` | Details about the outcome of the procedure.
+| *Mode* 3+| READ
+|===
+
+
+== Transaction management
+
+[[procedure_tx_getmetadata]]
+=== tx.getMetaData()
+
+
+.Details
+|===
+| *Syntax* 3+m| tx.getMetaData() :: (metadata)
+| *Description* 3+a| Provides attached transaction metadata.
+.2+| *Return arguments* | *Name* | *Type* | *Description*
+| `metadata` | `MAP` | Metadata about the transaction.
+| *Mode* 3+| DBMS
+|===
+
+
+[[procedure_tx_setmetadata]]
+=== tx.setMetaData()
+
+
+.Details
+|===
+| *Syntax* 3+m| tx.setMetaData(data)
+| *Description* 3+a| Attaches a map of data to the transaction. The data will be printed when listing queries, and inserted into the query log.
+.2+| *Input arguments* | *Name* | *Type* | *Description*
+| `data` | `MAP` | Metadata to attach to the transaction.
+| *Mode* 3+| DBMS
+|===
+
+
```

Comment:
So, now we don't have a table with all active procedures, do we? We have only 2 tables with removed and deprecated procedures respectively?

Reply:
Yes, that's correct. We have all procedures + deprecated and removed in case users are interested in what's deprecated in particular.

Reply:
I might be wrong but our test reports two views: summary (for a table) and detailed (for a list of procedures with descr)
So, now it fails because the table was removed